apps/condo/domains/miniapp/utils/b2bAppServiceUserAccess/server.utils.js

```diff
@@ -132,6 +132,41 @@ const canManageByServiceUser = async ({ authentication: { item: user }, listKey,
     return !isEmpty(B2BAppContexts)
 }
 
+const canExecuteByServiceUser = async (args, schemaConfig) => {
+    const { authentication: { item: user }, args: { data }, info: { fieldName } } = args
```

Comment:
data can be undefined in case of different args

Reply:
refactored